### PR TITLE
Added missing FASL file links from L; to LISP; directory.

### LINKED ISSUE #1288: Missing links in LIBLSP directory
L   HUMBLE FASL   LISP HUMBLE FASL
 L   LEDIT* FASL   LISP LEDIT FASL
 L   LET    FASL   LISP LET FASL

### PR DESCRIPTION
**Pull request by eswenson1**
Resolves #1288.

### CODE REVIEW

**Comment by larsbrinkhoff** [comment]
Is this due for review?

**Comment by eswenson1** [comment]
Did I forget to add you as a reviewer?

**Comment by eswenson1** [comment]
You don’t want to review this yourself, Lars? It’s so trivial!

**Comment by larsbrinkhoff** [comment]
Yes, there were no reviewers, so I added myself.  Then I thought, maybe it's not time to review yet?

File: build/lisp.tcl [hunk 1]
```diff
@@ -118,6 +118,10 @@ respond "_" "lisp;_nilcom;let\r"
 respond "_" "\032"
 type ":kill\r"
 
+respond "*" ":link l;humble fasl,lisp;\r"
+respond "*" ":link l;ledit* fasl,lisp;\r"
+respond "*" ":link l;let fasl,lisp\r"
```
Comment (atsampson) [inline build/lisp.tcl]: Should there be a ; at the end of this?

**Comment by eswenson1** [comment]
Yup. Sure should. I’ll fix

**Comment by eswenson1** [comment]
Teach me to call anything trivial.

**Pull request merged**